service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsControllerHearingInPastMidEventTest.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Map;
+
+import static java.lang.Long.parseLong;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ActiveProfiles("integration-test")
+@OverrideAutoConfiguration(enabled = true)
+@WebMvcTest(ManageHearingsController.class)
+public class ManageHearingsControllerHearingInPastMidEventTest extends AbstractControllerTest {
+
+    private static final String CASE_ID = "12345";
+
+    private LocalDateTime pastDate = LocalDateTime.now().minusDays(1);
+
+    ManageHearingsControllerHearingInPastMidEventTest() {
+        super("manage-hearings");
+    }
+
+    @Test
+    void shouldCorrectBothHearingDatesToCorrectOneWhenIncorrectAndPastHearingDatesIsEnabled() {
+        LocalDateTime correctStartDate = LocalDateTime.now().minusDays(2);
+        LocalDateTime correctEndDate = LocalDateTime.now().minusDays(2);
```

Comment:
Both are the same date, maybe end date should be LocalDateTime.now().minusDays(1)